src/components/OnboardingDialog/index.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react'
+import styled from '@emotion/styled'
+import PropTypes from 'prop-types'
+import { radius, shadow, space, duration, transition } from '../../theme'
+import { useTransition } from '../../hooks'
+
+const BodyWrapper = styled.div`
+  background-color: white;
+  border-radius: ${radius.lg};
+  box-shadow: ${shadow.strong};
+  padding: ${space[32]};
+`
+
+const OnboardingDialogBody = ({ children }) => (
+  <BodyWrapper>{children}</BodyWrapper>
+)
+
+OnboardingDialogBody.propTypes = {
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node,
+  ]).isRequired,
+}
+
+const DialogWrapper = styled.div`
+  position: absolute;
+  top: ${props => space[props.positionTop]};
+  left: 0;
+  right: 0;
+  z-index: 1;
+`
+
+const OnboardingDialog = ({ children, show, positionTop }) => {
+  if (!show) return null
+
+  return <DialogWrapper positionTop={positionTop}>{children}</DialogWrapper>
+}
+
+OnboardingDialog.defaultProps = {
+  positionTop: 56,
+}
+
+OnboardingDialog.propTypes = {
+  positionTop: PropTypes.number,
+  show: PropTypes.bool.isRequired,
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node,
+  ]).isRequired,
+}
+
+const Container = styled.div`
+  position: relative;
+`
+
+const Backdrop = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(255, 255, 255, 0.8);
```

Comment:
Dow we already have something similar, opacity-wise? 🤔 Or maybe this should be within the theme.

Reply:
I'll check it out.

Reply:
I'll add it to the theme. I'll call it `novaAlpha`.